packing-api.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# Script to pack api app for severless or micro contanier service
+echo 'Building ... \n' && yarn workspace @ehpr/api build && \
+echo 'Updating prod dependencies...\n' && yarn workspaces focus @ehpr/api --production  && \
+echo 'Deleting existing build dir\n' && rm -rf ./.build || true && \
+echo 'Creating build dir...\n' && mkdir -p .build/api && \
+echo 'Copy Node modules....\n' && cp -r node_modules .build/api && \
```

Comment:
Q: Does this rely on an empty node_modules at the start of the script?

Reply:
`yarn workspaces focus @ehpr/api --production` create node_modules for sure